libminifi/test/script-tests/ExecutePythonProcessorTests.cpp

```diff
@@ -54,13 +60,16 @@ class ExecutePythonProcessorTestBase {
   }
 
   std::string getScriptFullPath(const std::string& script_file_name) {
+    if (script_file_name.find('/') != std::string::npos || script_file_name.find("\\") != std::string::npos) {
```

Comment:
would we want this to be equivalent to `utils::file::resolve`?

Reply:
Good point, it's technically the same. Replaced it in f0cae0afc62088ebaa87610c297e79ee75fc7dc4